platform-sdk/docs/proposals/README.md

```diff
@@ -0,0 +1,145 @@
+contain diagrams and supporting materials. Diagrams must be created with DrawIO and exported in SVG format. All content
+must be editable (i.e. no images).
+
+If any APIs were developed in code as part of the design process, it may be included in the proposal directly or in the
+subdirectory of the proposal, but not merged into the source code of the platform.
+
+While in the draft phase the proposal may be updated and modified as needed. It is best practice to solicit review from
+stakeholders to surface any potential change requests while the proposal is in the draft phase. All grammar and spelling
+errors should be caught during this phase.
+
+### Creating a Draft Proposal
+
+1. Create a new branch for the proposal.
+2. Create a subdirectory of `platform-sdk/docs/proposals/` with the name of the proposal.
+3. Copy the template `template.md` into the subdirectory and give it an appropriate name.
+4. Fill out the template with the details of the proposal, removing any irrelevant sections or adding new sections as
+   needed. Diagrams and other files may be included in the proposal directory and linked from the proposal.
+5. Create a draft PR with the proposal documents.
+6. Put the draft PR in the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) board in
+   the `Draft` status.
+7. Solicit feedback from stakeholders and other design collaborators.
+8. Iterate on the proposal until it is ready for voting. All open comments should be resolved while in the draft stage.
+
+### Submitting a Proposal for Voting
+
+When the proposal is ready for voting, take the following steps:
+
+1. Ensure all comments are resolved.
+2. Mark the PR as `Ready for Review`
+3. Update the status of the PR in
+   the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) board to `Voting`
+
+Requests for immaterial changes can be addressed by pushing additional commits to the existing proposal
+PR while in `Voting`. Examples of immaterial changes are spelling or grammar errors, wording changes that do not modify
+meaning or intent, and formatting changes. Examples of material changes include changes to behaviors or APIs, addition
+or removal of diagrams, etc. If any material changes are needed, the PR must be closed and moved to the `Superseded`
+status. A new proposal PR is prepared and voting is restarted. The `Superseded` PR must reference the new PR to
+create an auditable history.
+
+---
+
+## Voting on A Proposal
+
+Votes follow the [Apache Voting](https://www.apache.org/foundation/voting.html#expressing-votes-1-0-1-and-fractions)
+scheme. Votes are cast in comments on the PR on a range between -1 to -0, +0 to +1 with the following semantics:
+
+* -1: is a veto.
+* -0 >= x > -1: is a vote generally against the proposal.
+* +0 <= x < +1: is a vote generally in favor of the proposal.
+* +1: is a vote in favor of the proposal.
```

Comment:
I'm still unconvinced that giving everybody veto power on everything is a good idea.

Reply:
Let's see how it works out. I don't think anyone will veto unless they feel quite strongly, and that will only happen if they really understand what the proposal is and means. Honestly, i'm more concerned about people voting +1 without having the full context. But that will sort itself out with some time.